src/actions/validators.js

```diff
@@ -0,0 +1,27 @@
+import { Jurisdiction } from '../contracts'
+import FetchingActions from './fetching'
+import * as ActionTypes from '../actiontypes'
+import { JURISDICTION_ADDRESS } from '../constants'
+import Network from "../network";
+
+const ValidatorsActions = {
+
+  findAll() {
+    return async function(dispatch) {
+      const jurisdiction = await Jurisdiction.at(JURISDICTION_ADDRESS)
+      const events = jurisdiction.ValidatorAdded({}, { fromBlock: 0, toBlock: 'latest' });
```

Comment:
This is a very slow request, we should look into it later. The jurisdiction should probably have a method to return the array of validators.

Reply:
I reported a bug and added a comment.